service/src/main/java/uk/gov/hmcts/reform/fpl/model/CaseData.java

```diff
@@ -275,7 +275,7 @@ public DynamicList buildRespondentDynamicList() {
     @JsonProperty("documents_socialWorkEvidenceTemplate_document")
     @Valid
     public final Document socialWorkEvidenceTemplateDocument;
-    public final CourtBundle courtBundle;
+    public final List<Element<CourtBundle>> courtBundle;
```

Comment:
this looks rather concerning, have you thought about how this is going to interact with old cases? It seems like it would probably break

Reply:
unless it is a temporary field, in which case ignore me

Reply:
it doesn't look like this field is used, could it just be removed?